### PR TITLE
Create physiopy-documentation.tex

### PR DESCRIPTION
**Pull request by goodalse2019**
Adding summary tex file for physiopy BrainHack project

### CODE REVIEW

File: summaries/physiopy-documentation.tex [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+If you want to add a citation, the \emph{cleveref} package is imported in \emph{main.tex} with the APA 6e style. If you want to add a citation, you can do so with the \verb|\textcite| and \verb|\parencite| commands: \\
+Parenthesis citation of some paper \parencite{Gau2021} and in text citation of the same paper from \textcite{Gau2021}.
+
+Print the bibliography at the end, and add the bibliography resource file (\emph{.bib} with a specific name) in the \emph{references} folders, importing it in the preamble of the main file (around line 77).
+Finally, package everything and send it over via email, or open a PR to the git repository.
+
+\end{comment}
```
Comment (smoia) [inline summaries/physiopy-documentation.tex]:
```suggestion
```

File: summaries/physiopy-documentation.tex [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.5\textwidth]{physiopy-documentation_figure.png}
+	\caption{Left: Current version of the documentation homepage; Right: Physiopy Contributors}
+	% Add a label to reference in text. Make it specific!
+	\label{fig:physiopy_beforeafter}
+\end{figure}
+
+-----------------------------------
```
Comment (smoia) [inline summaries/physiopy-documentation.tex]:
```suggestion
```

**Review by smoia — APPROVED** [review]
LGTM!

**Comment by htwangtw** [comment]
Merging :+1:

**Pull request merged**